src/tools/pontoon/search_filters.md

```diff
@@ -2,53 +2,56 @@
 
 <!-- toc -->
```

Comment:
Looking at the rendered page and given the size of content, I think we can lose this.

Reply:
Removed.